joomla/joomla-cms
# [com_plugins] Complete search tooltip

#### Summary of Changes

Very simple PR to improve the search tooltip and element label (accessibility) in com_plugins.
#### Testing Instructions

Code review, or:
1. Apply patch
2. Go to Extensions -> Plugins, check the tooltip in the search field.
